pkgs/cli/src/main.zig

```diff
@@ -28,5 +42,34 @@ pub fn main() !void {
 
             try clock.run();
         },
+        .prove => |provecmd| {
+            const state = types.BeamState{
```

Comment:
i guess i can replace it by genMockChain

Reply:
feel free to